src/ontology/uberon-edit.obo

```diff
@@ -177321,7 +177322,7 @@ alt_id: UBERON:0025487
 synonym: "regional part of gustatory epithelium" EXACT HUMAN_PREFERRED [BIRNLEX:4100]
 is_obsolete: true
 consider: BIRNLEX:4100
-consider: http://uri.neuinfo.org/nif/nifstd/nifext_13
```

Comment:
@balhoff, do you also know this one? 👀

Reply:
I found the curie nifext:13 in the Bioregistry, and it redirects to [this page](https://scicrunch.org/scicrunch/interlex/view/ilx_0109850), which shows that the preferred prefix is NIFEXT, but OWL API compressed the URI to `nifext:13`. Is this way correct, and I don't need to define it?